uat/custom-components/client-java-paho/src/main/java/com/aws/greengrass/testing/mqtt5/client/paho/MqttConnectionImpl.java

```diff
@@ -197,15 +203,16 @@ public MqttPublishReply publish(long timeout, @NonNull Message message) {
             logger.atError().withThrowable(ex)
                     .log("Failed during publishing message with reasonCode {} and reasonString {}",
                             ex.getReasonCode(), ex.getMessage());
-            builder.setReasonCode(ex.getReasonCode());
-            builder.setReasonString(ex.getMessage());
+            throw new RuntimeException(ex.getMessage(), ex);
```

Comment:
The same as before.
1. Message of other throw is reused
2. Why RuntimeException if method define will throws MqttException ?